sdk/tests/js/webgl-test-utils.js

```diff
@@ -739,7 +739,16 @@ var glTypeToTypedArrayType = function(gl, type) {
     case gl.INT:
       return window.Int32Array;
     case gl.UNSIGNED_INT:
+    case gl.UNSIGNED_INT_5_9_9_9_REV
+    case gl.UNSIGNED_INT_10F_11F_11F_REV:
+    case gl.UNSIGNED_INT_2_10_10_10_REV:
+    case gl.UNSIGNED_INT_24_8:
       return window.Uint32Array;
+    case gl.HALF_FLOAT:
```

Comment:
Sorry I didn't catch this in the first round, but you will want to add HALF_FLOAT_OES (since it requires an extension to access the enum, it's probably better to just use the number with a comment saying it's HALF_FLOAT_OES).